sonarts-core/package.json

```diff
@@ -58,7 +58,7 @@
     "@types/vis": "4.21.0",
     "awesome-typescript-loader": "3.3.0",
     "glob": "7.1.2",
-    "jest": "21.2.1",
+    "jest": "^23.5.0",
```

Comment:
Do we need to update "jest"? Anyway we should have fixed version (without ^).

Reply:
agreed, this should not be required.